src/Http/Requests/SyncModelPropertyValues.php

```diff
@@ -45,4 +46,11 @@ public function authorize()
     {
         return true;
     }
+
+    public function messages()
+    {
+        return [
+            'propertyValues.*.integer' => __('Each property value must be a valid integer.'),
```

Comment:
From the user's perspective, it's not an integer because they need to pick a valid entry from the dropdown
```suggestion
            'propertyValues.*.integer' => __('Each property value must be a valid entry.'),
```